numpy/_core/tests/test_ufunc.py

```diff
@@ -616,6 +616,31 @@ def call_ufunc(arr, **kwargs):
         expected = call_ufunc(arr.astype(np.float64))  # upcast
         assert_array_equal(expected, res)
 
+    @pytest.mark.parametrize("ufunc", [np.add, np.equal])
+    def test_cast_safety_scalar(self, ufunc):
+        # We test add and equal, because equal has special scalar handling
+        # Note that the "equiv" casting behavior should maybe be considered
+        # a current implementation.
```

Comment:
```suggestion
        # a current implementation detail.
```

Not sure if this is what you meant or not

Reply:
thanks, yes, I mean to say: I don't care if future someone thinks they know better and wants to change it.